demo.html

```diff
@@ -90,10 +92,39 @@
               size: 128,
               format: 'svg'
             };
-
+            // Default base64
             data = new Identicon(hash, options).toString();
             document.write(
-              '<br><img src="data:image/svg+xml;base64,' + data + '"><br><br>'
+              '<br>svg+xml;base64 (1280 bytes)<br><img src="data:image/svg+xml;base64,' + data + '"><br><br>'
+            );
+            </script>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="col-md-12">
+          <h4>Raw</h4>
+          <p>Identicon toString method accepts a raw argument that when set to true returns a XML-based SVG element.</p>
```

Comment:
Hesitant to use the word unencoded because in the case of SVG it is encoded as utf8. Understand this may not be case for the getDump() when using png; thinking we should use the word raw to cover our bases.